mapper.go

```diff
@@ -58,7 +59,7 @@ func (m *metricMapper) initFromString(fileContents string) error {
 
 		switch state {
 		case SEARCHING:
-			if line == "" {
+			if line == "" || line[0] == COMMENT_CHAR {
```

Comment:
Give the example above, if I skipped the line outside the switch, I would get an error because I was still in METRIC_DEFINITION state.